### PR TITLE
Alert when the playground services no requests for a while

### PR DESCRIPTION
**Pull request by shepmaster**

### CODE REVIEW

File: ansible/playbooks/monitoring.yml [hunk 1]
```diff
@@ -244,6 +244,15 @@
                 summary: "Self-Hosted runner {{ '{{ $labels.runner }}' }} in repo {{ '{{ $labels.repo }}' }} is offline"
                 description: "GitHub is reporting that the self-hosted runner {{ '{{ $labels.runner }}' }} in the repo {{ '{{ $labels.repo }}' }} is offline. This could be caused by the machine hosting the runner having problems, or by the runner software being out of date."
 
+        - name: playground
+          rules:
+            - alert: PlaygroundDown
+              expr: sum(irate(playground_request_duration_seconds_count[5m])) == 0
```
Comment (pietroalbini) [inline ansible/playbooks/monitoring.yml:250]:
```suggestion
              expr: sum(increase(playground_request_duration_seconds_count[5m])) == 0
```

Comment (shepmaster) [reply ansible/playbooks/monitoring.yml]: I'll have to figure out the meaningful difference here I suppose.

**Comment by shepmaster** [comment]
Is there an easy way to make the Zulip notification for the playground alert have an `@` mention for me?

**Comment by pietroalbini** [comment]
> Is there an easy way to make the Zulip notification for the playground alert have an @ mention for me?

Unfortunately no.

**Pull request merged**